index.html

```diff
@@ -8,6 +8,69 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+    <div id="blue-box">
+        <div id="join-us">
+            <div id="subscribe">
+                    <div class="msg">Subscribe to get information, latest news and other</div>
+                    <div class="msg">interesting offers about Cobham</div>
+            </div>
+            <div id="mail">
+                <div id="mail-bar">
+                        <img id="mail-image" src="./images/email.svg">
+                        <div id="sub-gap">Your email</div>
+                </div>
+                <div id="button">
+                    <div id="green">Subscribe</div>
+                </div>
+            </div>
+        </div>
+        <img id="telegram" src="./images/tele.svg" />
+    <!-- <img id="stars" src="./images/stars.svg" /> -->
+    </div>
+    <div id="second-half">
+        <div id="social-media">
+            <div id="travel-go">
+                <div id="goo">
+                    <div class="headg">Travel</div>
+                    <div id="tg" class="headg">Goo</div>
+                </div>
+                <div class="grey">Book your ticket in minute, get full</div>
+                <div class="grey">control for much longer.</div>
+            </div>
+            <div id="about">
+                <div id="Contact">
+                    <div class="headin">Company</div>
+                    <div class="gap">About</div>
+                    <div class="gap">Careers</div>
+                    <div class="gap">Mobile</div>
+                </div>
+                <div id="Contact">
+                    <div class="headin">Contact</div>
```

Comment:
remove unnecessary and repetitive classes and ids and use css combinators instead